js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -419,6 +420,21 @@ export class MiniAppBridge {
       );
     });
   }
+
+  /**
+   * Associating get point balance function to MiniAppBridge object.
+   * (provided rakuten.miniapp.user.POINTS is allowed by the user)
+   */
+  getPoints() {
+    return new Promise<Points>((resolve, reject) => {
+      return this.executor.exec(
+        'getPoints',
+        null,
+        points => resolve(JSON.parse(points) as Points),
+        error => reject(error)
```

Comment:
I think for new functionality, we want to switch to using `MiniAppError` for errors instead of just passing a string. See `getAccessToken` above for an example. But I think in the case of `getPoints`, there aren't any specific error types which we want to support. So maybe just creating `MiniAppError` is enough?

@Climbatize @rleojoseph @khairul-alam-licon @munir-rakuten What do you think on this?

Reply:
@corycaywood 👍 I've put default case with MiniAppError

```
...
switch (errorType) {
              default:
                return reject(new MiniAppError(miniAppError));
            }
```